drivers/gasket/manifest.yaml

```diff
@@ -0,0 +1,12 @@
+version: v1alpha1
+metadata:
+  name: gasket
+  version: "$VERSION"
+  author: Sidero Labs
+  description: |
+    This system extension provides google gasket driver kernel modules built against a specific Talos version.
+    This driver is required for PCIe and M.2 Google Coral accelerators. There are 2 kernel modules ("gasket"
+    and "apex") required to enable this driver.
+  compatibility:
+    talos:
+      version: ">= v1.2.0"
```

Comment:
For kernel modules extensions, should the version be constrained to the only the precise Talos version? My understanding is that each Talos version would need to have a distinct version of the kernel modules because the signature would be different for each Talos version.

Reply:
this is fine, since the module would support any talos versions matching this constraint, the signature is not something that would prevent the module from installing in the first place